src/statement.rs

```diff
@@ -217,6 +217,12 @@ impl<'l> Statement<'l> {
         &self.column_names
     }
 
+    /// Return column names (wrapped in Rc).
+    #[inline]
+    pub(crate) fn column_names_rc(&self) -> Rc<Vec<String>> {
```

Comment:
I think you can just tag the field as `pub(crate)` and clone it without this method.

Reply:
Done